### ISSUE TITLE
package.json missed

### ISSUE THREAD

**Issue by johnwangwjq** [ryannhg/elm-spa-realworld#4]
Hi @ryannhg , 

How to run the app? I started it by `npm start` but failed with `missing package.json`. I'm new to npm and elm. How to create the required package.json?

**Comment by rosensama** [comment]
Seems it's missing other code, too.  I don't see source for Effect, Request, etc. which maybe are in the missing directories?

```
$ elm make
Dependencies ready!         
-- MISSING SOURCE DIRECTORIES ----------------------------------------- elm.json

I need a valid elm.json file, but the "source-directories" field lists the
following directories:

    .elm-spa/defaults
    .elm-spa/generated

I cannot find them though. Are they missing? Are there typos?
```

**Comment by rosensama** [comment]
Yep, this is the migration to the elm-spa 6.x betas.   To run the example locally:
1) `npm install elm-spa@6.0.15--beta -g`
2) `elm-spa server`

**Comment by setrar** [comment]
> Yep, this is the migration to the elm-spa 6.x betas. To run the example locally:
> 
>     1. `npm install elm-spa@6.0.15--beta -g`
> 
>     2. `elm-spa server`

Thanks @rosensama, I almost gave up on elm-spa

I tried following the [`a realworld app with elm-spa`](https://rhg.dev/posts/realworld-elm-spa) blog which I found so interesting.

Coming from a CLI world where I enjoy using:

```
$ ng generate component ... --skip-tests 
```

Stumbling on a version issue is somewhat embarassing

```
$ npm install elm-spa@latest --global
```

:bulb: For others having the same issue, I use

```
$ npm --version
6.14.12
```

```
$ node --version
v14.16.1
```

```
$ elm-spa --version

elm-spa – version 6.0.15--beta

Commands:
elm-spa new . . . . . . . . .  create a new project
elm-spa add <url> . . . . . . . . create a new page
elm-spa build . . . . . . one-time production build
elm-spa watch . . . . . . .  runs build as you code
elm-spa server  . . . . . . start a live dev server

Visit https://elm-spa.dev for more!
```

on a `M1 Silicon`

<img width="268" alt="Screen Shot 2021-05-17 at 9 04 03 AM" src="https://user-images.githubusercontent.com/123704/118493490-2b9bc580-b6ef-11eb-832c-0a325ddf1cae.png">
